quoting the parameters doesn't work.  this is `sh` which lacks the necessary array functionality.

the wiki for [this rule](https://github.com/koalaman/shellcheck/wiki/SC2086) suggests this use case is an acceptable if unhappy exception.

This needs to be a non-quoted string and should have the comment to prevent shellcheck failures on this line.

--- a/cluster/addons/fluentd-elasticsearch/fluentd-es-image/run.sh
+++ b/cluster/addons/fluentd-elasticsearch/fluentd-es-image/run.sh
@@ -20,4 +20,8 @@
 # For systems without journald
 mkdir -p /var/log/journal
 
+# Use exec to get the signal
+# A non-quoted string and add the comment to prevent shellcheck failures on this line.
+# See https://github.com/koalaman/shellcheck/wiki/SC2086
+# shellcheck disable=SC2086
 exec /usr/local/bin/fluentd $FLUENTD_ARGS

--- a/hack/.shellcheck_failures
+++ b/hack/.shellcheck_failures
@@ -1,7 +1,6 @@
 ./build/lib/release.sh
 ./cluster/addons/addon-manager/kube-addons.sh
 ./cluster/addons/fluentd-elasticsearch/es-image/run.sh
-./cluster/addons/fluentd-elasticsearch/fluentd-es-image/run.sh
 ./cluster/clientbin.sh
 ./cluster/common.sh
 ./cluster/gce/config-common.sh

